### ISSUE TITLE
VTX support

### ISSUE THREAD

**Issue by digitalentity** [iNavFlight/inav#382]
Support for boards with built-in VTX (SirinFPV)

**Comment by digitalentity** [comment]
Blocked by #338

**Comment by DzikuVx** [comment]
@digitalentity is this still relevent? We have SmartAudio and Tramp support, that more less covers VTX supprt nowadays

**Comment by digitalentity** [comment]
No, this is no longer relevant. Closing
